pkg/resources/resources.go

```diff
@@ -713,6 +713,9 @@ const (
 	ExternalAKSClusterSubscriptionID  = "subscriptionID"
 	ExternalAKSClusterClientID        = "clientID"
 	ExternalAKSClusterClientSecret    = "clientSecret"
+	AKSNodepoolNameLabel              = "kubernetes.azure.com/agentpool"
+	EKSNodeGroupNameLabel             = "eks.amazonaws.com/nodegroup"
+	GKENodepoolNameLabel              = "cloud.google.com/gke-nodepool"
```

Comment:
Did _we_ define those constants? If so, we should probably not squat other company's domains.

Reply:
Seems like we do not, those are upstream labels (e.g. for GKE https://cloud.google.com/kubernetes-engine/docs/concepts/node-pools ).